projects/models.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  pkgs,
+  sources,
+}:
+let
+  yants = import sources.yants { };
+
+  inherit (yants)
+    string
+    list
+    option
+    attrs
+    enum
+    either
+    ;
+in
+rec {
+  project =
+    p: with p; {
+      name = string name;
+      metadata = with metadata; {
+        summary = string summary;
+        funds = list (enum [
+          "Commons"
+          "Core"
+          "Entrust"
+          "Review"
+        ]) funds;
```

Comment:
Maybe leave funds out as long as we don't have code/processes for adding values. As discussed elsewhere, that information is attached to subgrants, and we want those separate anyway so we can link to https://nlnet.nl/project.